ksp_plugin/vessel.hpp

```diff
@@ -183,8 +185,9 @@ class Vessel {
   // |desired_t_min|.
   void RequestReanimation(Instant const& desired_t_min) EXCLUDES(lock_);
 
-  // Blocks until the |t_min()| of the vessel is at or before |desired_t_min|.
-  void WaitForReanimation(Instant const& desired_t_min) EXCLUDES(lock_);
+  // Same as |RequestReanimation|, but synchronous.  This function blocks until
+  // the |t_min()| of the vessel is at or before |desired_t_min|.
+  void AwaitReanimation(Instant const& desired_t_min) EXCLUDES(lock_);
```

Comment:
Is this only used in VesselTest?

Reply:
At the moment yes, but I could imagine cases where we would need to actually wait for a vessel reanimation.  So it seems useful to have the entry point implemented and tested, instead of discovering later that we need it.